assets/js/modules/analytics/components/common/ProfileSelect.js

```diff
@@ -117,7 +117,10 @@ export default function ProfileSelect( { hasModuleAccess } ) {
 		);
 	}
 
-	if ( ! ga4ReportingEnabled ) {
+	if (
+		! ga4ReportingEnabled &&
+		! profiles.find( ( profile ) => profile.id === PROFILE_CREATE )
```

Comment:
Hey @jimmymadon - I feel a bit pedantic for this, but we do tend to use `some()` in the codebase which returns a boolean, when we don't actually need the array element result of `find()`. So, I'd suggest updating as follows:

```suggestion
		! profiles.some( ( profile ) => profile.id === PROFILE_CREATE )
```